### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by saneera** [fabioformosa/quartz-manager#3]
HI 

thanks for the application working fine , anyway is there any possibility to add multiple jobs 

Thanks

**Comment by fabioformosa** [comment]
No, I'm sorry, this feature is in the roadmap, but  at the moment it supports only single job.
I suggest you to deploy multiple quartz-managers to launch multiple jobs.

**Comment by fabioformosa** [comment]
Just in progress and pushed in a feature branch. Soon it will reviewed and merged into the master and release a new major version

**Comment by fabioformosa** [comment]
merged in master and released v5.0.1.
